docs/submission/submission-overview.md

```diff
@@ -13,17 +13,17 @@ Participating programs submit clinical data through the **[ARGO Data Platform](h
 
 ## Data Dictionary
 
-To support the gathering of high-quality data, a clinical data dictionary has been developed that performs rigorous validation on submitted data at the time of submission. Clinical data is defined in logical groupings, which will be submitted as TSV files in the ARGO Program Services section.
+To support the gathering of high-quality data, a clinical data dictionary has been defines the ARGO data model, as well as some data validations that are performed at the time of submission. Clinical data is defined in logical groupings, which will be submitted as TSV files in the ARGO Program Services section.
 
 Explore the details of the ARGO clinical dataset in the **[ARGO Data Dictionary Viewer](/dictionary).**
 
 ## Data Submission and Release Process
 
 The ARGO Data Platform has been optimized to ensure that clinical and molecular data upload is intuitive and efficient for data submitters. ARGO data submission and release happens in 4 major steps:
 
-1. **Register Molecular Samples:** [Registering samples](registering-samples) within the Program Services section in the ARGO Data Platform initializes the ARGO primary identifiers that will be assigned to submitted data entities. Registering samples with specimen and donor identifiers upfront maintains data integrity across the data submission and processing pipelines.
+1. **Register Molecular Samples:** [Registering samples](registering-samples) within the Program Services section in the ARGO Data Platform initializes the ARGO primary identifiers that will be assigned to submitted data entities. Registering samples and associated specimen and donor identifiers upfront maintains data integrity across the data submission and processing pipelines.
 1. **Submit Donor Clinical and Molecular Data:** Clinical Data Submitters can use the Program Services section in the ARGO Data Platform to submit clinical data. [Submitting clinical data](submitting-clinical-data) is easy and intuitive, facilitated by a guided clinical submission interface. In parallel, Molecular Data Submitters can begin submitting molecular data. [Submitting molecular data](submitting-molecular-data), using the Song and Score clients, is fast and secure.
-1. **Molecular Data Processing:** Once raw molecular data has been submitted, [analytic workflows](../analysis-workflows/dna-pipeline) will be automatically kicked off for uniform analysis of all donor samples.
+1. **Molecular Data Processing:** Once raw molecular data has been submitted, [analytic workflows](../analysis-workflows/analysis-overview) will be automatically kicked off for uniform analysis of all donor samples.

```

Comment:
the links in this section seem to be broken on https://content--argo-docs.netlify.app/docs/submission/submission-overview/
- can you double check all them?

Reply:
they all worked on my local docs site?

Reply:
Ciaran is looking into this...I may need to go in and change all links because I'm seeing them broken in some cases